api/go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/ceph/ceph-csi-operator/api
```

Comment:
i think you need to update the go.mod in the root folder to use this as a replace section. can you please check

Reply:
updated the PR, closing this conversation